src/emu.rs

```diff
@@ -183,7 +188,7 @@ impl Emu {
             break_on_next_cmp: false,
             break_on_next_return: false,
             filename: String::new(),
-            enabled_ctrlc: true,
+            enabled_ctrlc: false, // TODO: make configurable with command line arg
```

Comment:
make this configurable via clap flags? it gets in the way when "quick iterating" developing.